zkevm-circuits/src/evm_circuit/execution/sstore.rs

```diff
@@ -467,636 +450,209 @@ impl<F: Field> SstoreTxRefundGadget<F> {
 
-    fn mock_tx() -> eth_types::Transaction {
-        let from = address!("0x00000000000000000000000000000000000000fe");
-        let to = address!("0x00000000000000000000000000000000000000ff");
-        eth_types::Transaction {
-            from,
-            to: Some(to),
-            ..Default::default()
-        }
-    }
+    use eth_types::{bytecode, Word};
+    use mock::{test_ctx::helpers::tx_from_1_to_0, TestContext, MOCK_ACCOUNTS};
 
     #[test]
-    fn sstore_gadget_warm_persist() {
+    fn sstore_gadget_no_refund() {
         // value_prev == value
         test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0x060504.into(),
-            0x060504.into(),
-            true,
-            true,
-        );
-        // value_prev != value, original_value == value_prev, original_value != 0
-        test_ok(
-            mock_tx(),
             0x030201.into(),
             0x060504.into(),
-            0x060505.into(),
-            0x060505.into(),
-            true,
-            true,
-        );
-        // value_prev != value, original_value == value_prev, original_value == 0
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
             0x060504.into(),
-            0.into(),
-            0.into(),
-            true,
-            true,
-        );
-        // value_prev != value, original_value != value_prev, value != original_value
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0x060505.into(),
-            0x060506.into(),
-            true,
-            true,
-        );
-        // value_prev != value, original_value != value_prev, value == original_value
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0x060505.into(),
             0x060504.into(),
-            true,
-            true,
         );
     }
 
-    fn sstore_gadget_warm_revert() {
-        // value_prev == value
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0x060504.into(),
-            0x060504.into(),
-            true,
-            false,
-        );
-        // value_prev != value, original_value == value_prev, original_value != 0
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0x060505.into(),
-            0x060505.into(),
-            true,
-            false,
-        );
-        // value_prev != value, original_value == value_prev, original_value == 0
-        test_ok(
-            mock_tx(),
-            0x030201.into(),
-            0x060504.into(),
-            0.into(),
-            0.into(),
-            true,
-            false,
-        );
-        // value_prev != value, original_value != value_prev, value != original_value
+    #[test]
+    fn sstore_gadget_case_a() {
```

Comment:
Can we call it `sstore_gadget_case_CLEARS_SCHEDULE_cost` or something similar?? (Ignore CAPS)

Reply:
done